BLL/Exceptions/AlreadyExist/CategoryAlreadyExistException.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BLL.Exceptions.AlreadyExist;
+
+public class CategoryAlreadyExistException : AlreadyExistException
```

Comment:
you do not need to create specific exceptions, it is enough to create one AlreadyExistException and call it in the services and send a message there